src/ports.js

```diff
@@ -0,0 +1,15 @@
+function Port (name) {
+    this.ports = name;
+    this.ships = []
+};
+
+Port.prototype = {
+    addShip(ship) {
+    this.ships.push(ship)   
```

Comment:
Indentation inside a function:
```suggestion
        this.ships.push(ship)
```